### ISSUE TITLE
Unable to build facetracker2osc on xcode

### ISSUE THREAD

**Issue by traumverloren** [ml4a/ml4a-ofx#34]
I am curious how I can fix this error I get when I try to build the facetracker2osc module? I get the following error:
`No matching member function for call to 'startThread'`  for `openFrameworks/addons/ofxControl/src/ofxControlBpm.cpp`

<img width="1211" alt="screenshot 2018-12-16 at 19 38 08" src="https://user-images.githubusercontent.com/9959680/50057416-743fa500-016a-11e9-933f-b26a02f81820.png">

i'm a web dev, but not very experienced with c++ or openFrameworks. So i apologize if I'm missing something simple. I'm on macos Mojave & xcode Version 10.1, if that matters. btw, i'm really enjoying your ml4a course and trying to get a project started myself. :)

**Comment by genekogan** [comment]
yes this is broken right now because the threading functions have been switched in OF since 0.10. we need to fix this. in the meantime it should still compile with OF 0.9 or you can get a binary in the latest release.

**Comment by traumverloren** [comment]
@genekogan Thanks for the reply. I was unable to get it to build in OF 0.97 (even with addressing this issue: https://forum.openframeworks.cc/t/the-i386-architecture-is-deprecated-on-xcode/30481/3), but the binary works! thanks so much! <3
